mino/minows/address_test.go

```diff
@@ -0,0 +1,232 @@
+package minows
+
+import (
+	"github.com/libp2p/go-libp2p/core/peer"
+	ma "github.com/multiformats/go-multiaddr"
+	"go.dedis.ch/dela/mino"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	AddrAllInterface = "/ip4/0.0.0.0/tcp/80"
+	AddrLocalhost    = "/ip4/127.0.0.1/tcp/80"
+	AddrHostname     = "/dns4/example.com/tcp/80"
```

Comment:
Please prepend all of these with `p2p://`, so it's `p2p://ipv4/0.0.0.0/tcp/80`. This will make it more similar to an URL.

Reply:
multiaddress requires a certain format. [this doc](https://docs.libp2p.io/concepts/fundamentals/addressing/) contains more details.